ads/_config.js

```diff
@@ -734,6 +734,11 @@ export const adConfig = {
     ],
     consentHandlingOverride: true,
   },
+  
+  'performancenative': {
+    prefetch: 'https://api.performancenative.com/assets/pn.js',
+    renderStartImplemented: true,
```

Comment:
I don't see this api being called when I run locally. Are you calling this method in your js? If not, you should remove this as it will delay rendering.